android-lib/src/main/java/com/kinvey/android/KinveyHeaders.java

```diff
@@ -27,13 +27,17 @@
 class KinveyHeaders extends com.kinvey.java.core.KinveyHeaders {
 
     @Key("x-kinvey-device-information")
+    private String deviceInformation;
+
```

Comment:
Yes, we are.

Reply:
Ok, makes sense for now, since we do not want to break any existing logic that may use information from the old header. But we should consider deprecating this in the future, and encourage developers to use the new header.

Reply:
I vote for doing this very soon. Maybe next release after we add the new header. I don't expect any impact elsewhere.